docs/start/ingest.md

```diff
@@ -0,0 +1,35 @@
+(start-ingest)=
+# Ingesting data
+
+:::{card} All data ingestion methods for CrateDB at a glance 
```

Comment:
Should this card be moved to the bottom to basically act as "Next Step"?

Reply:
Sure. Fixed with 1123249e64.